internal/apis/certmanager/types_issuer.go

```diff
@@ -292,6 +292,12 @@ type CAIssuer struct {
 	// certificate will be issued with no OCSP servers set. For example, an
 	// OCSP server URL could be "http://ocsp.int-x3.letsencrypt.org".
 	OCSPServers []string
+
+	// IssuingCertificateURLs is a list of URLs which this issuer should embed into certificates
+	// it creates. See https://www.rfc-editor.org/rfc/rfc5280#section-4.2.2.1 for more details.
+	// As an example, such a URL might be "http://ca.domain.com/ca.crt".
+	// +optional
+	IssuingCertificateURLs []string `json:"issuingCertificateUrls,omitempty"`
```

Comment:
suggestion: The casing in the JSON here doesn't match the style we [generally use](https://google.github.io/styleguide/go/decisions#initialisms) in our code and it would catch me out trying to use this feature!

Could we use the following instead?

```suggestion
	IssuingCertificateURLs []string `json:"issuingCertificateURLs,omitempty"`
```

Reply:
Done